doc/index.rst

```diff
@@ -9,11 +9,13 @@ lsst.ts.ess
 ``ts_ess`` is a Commandable SAL Component (CSC) to control various environmental sensors at the Vera C. Rubin Observatory.
 There will be at least one, and likely several, CSC(s) running at Rubin Observatory.
 The CSC makes a socket connection to several Raspberry Pi 4's, to which the actual sensors ar connected, distributed around the observatory.
-The Raspberry Pi 4's then use the code from `ts_envsensors`_ to retrieve the sensor telemetry from the sensors and to convert the telemetry to a common format.
+The Raspberry Pi 4's then use the code from `ts_ess_sensors`_ to retrieve the sensor telemetry from the sensors and to convert the telemetry to a common format.
+The two projects share common code in `ts_ess_common` _.
 The telemetry then is sent to the CSC which sends it to the EFD.
-The protocol used for commands for controlling the ``ts_envsensors`` code and well for transferring the telemetry is described in :doc:`Communication Protocols <protocols>`.
+The protocol used for commands for controlling the ``ts_ess_sensors`` code and well for transferring the telemetry is described in :doc:`Communication Protocols <protocols>`.
```

Comment:
Minor existing point at line 14: the CSCs don't directly send data to the EFD. They publish data as SAL topics. Another service forwards the data to the EFD. I'm not sure if this is worth correcting.

Reply:
Thanks, I corrected that and I removed the word `well` from line 15.